angular-app/src/app/user-profile/user-profile.component.html

```diff
@@ -31,40 +38,33 @@
       </mat-panel-description>
     </mat-expansion-panel-header>
     <mat-list role="list">
-      <mat-list-item role="listitem">Sunny Day</mat-list-item>
-      <mat-list-item role="listitem">Fall Sunset</mat-list-item>
-      <mat-list-item role="listitem">Cumulus Clouds</mat-list-item>
+      <mat-list-item role="listitem" *ngFor= "let snippet of profile.snippets">
+        {{snippet}}
+      </mat-list-item>
     </mat-list>
   </mat-expansion-panel>
 
-<div> Tags:
-  <mat-chip-list aria-label= "Tags">
-    <mat-chip>Country</mat-chip>
-    <mat-chip>Guitar</mat-chip>
-    <mat-chip>Jazz</mat-chip>
-    <mat-chip>Producer</mat-chip>
-  </mat-chip-list>
-</div>  
+</mat-card>
 
 <div> Posts:
-<mat-card class= "profile-post">
+  <mat-card class= "profile-post">
```

Comment:
Can the mock post be deleted since the User is being authenticated from Firebase?

Reply:
Posts aren't connected to users yet, which is why I left it.